oval-schemas/windows-definitions-schema.xsd

```diff
@@ -1485,7 +1485,17 @@
       <xsd:element name="ntuser_object" substitutionGroup="oval-def:object">
             <xsd:annotation>
                   <xsd:documentation>The ntuser_object element is used to specify which metadata should be collected from a Windows ntuser.dat file.  Each object extends the standard ObjectType as defined in the oval-definitions-schema and one should refer to the ObjectType description for more information. The common set element allows complex objects to be created using filters and set logic. Again, please refer to the description of the set element in the oval-definitions-schema.</xsd:documentation>
-                  <xsd:appinfo>
+                  <xsd:documentation>To ensure consistent results across OVAL interpreters should comply with the following:</xsd:documentation>
+				  <xsd:documentation>1.  Only include ntuser.dat files from 'human' accounts that have interactively logged into the system, with the exception of the 'Default' user if the 'include_default' behavior is true.</xsd:documentation>
+				  <xsd:documentation>2.  Create a 'does not exist' item for each ntuser.dat file that does contain the required registry key.</xsd:documentation>
```

Comment:
Should say "does not contain", I think.

Reply:
Although... you do know this requirement cannot impact test results, and therefore, should be more of a strong suggestion than a requirement.

Reply:
Agreed on "does not contain", good catch.   As the overall documentation is a suggestion, I think we are OK?

To ensure consistent results across OVAL interpreters **should** comply with the following:

Reply:
I think I've resolved this feedback item?